CMakeLists.txt

```diff
@@ -34,10 +34,16 @@ add_executable (unittest
     test/tests.cpp
     )
 
-target_link_libraries (unittest
-    qrack
-    pthread
-    )
+if (MSVC)
+    target_link_libraries (unittest
+        qrack
```

Comment:
Can we pull these out to a single variable specified somewhere so we don't have to duplicate these for each call?

Reply:
Done.